.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create an issue at https://projects.theforeman.org/projects/foreman/issues
+* Reference the issue via "Fixes #1234" or "Refs #1234" in the commit message
+* Extract all strings for i18n, see Translating section in the guide
```

Comment:
You can format is as a markdown link

Reply:
But markdown does not apply here, this is shown in the HTML edit textarea and the text is supposed to be deleted. Markdowning it would make it less readable.

Reply:
Right, I was confused by the .md extension (compared to contributing.md).

Reply:
Yeah it is actually markdown, I just think links would be overkill.